website/docs/kubernetes/openshift/configuring-access-to-a-developer-sandbox.md

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 1
+title: Developer Sandbox
+description: Configuring access to a Developer Sandbox
+keywords: [podman desktop, podman, containers, pods, migrating, kubernetes, openshift]
+tags: [migrating-to-kubernetes, openshift]
+---
+
+# Configuring access to a Developer Sandbox
+
+The [Developer Sandbox](https://developers.redhat.com/developer-sandbox) is a free, private OpenShift environment including one project and a resource quota of 14 GB RAM, and 40 GB storage.
```

Comment:
The FAQ (https://developers.redhat.com/developer-sandbox/FAQ) has different values. Where did you get them from ?

Reply:
I copied this from somewhere. If not this page, something displayed by the extension.

Reply:
Sandbox team confirmed their FAQ is not up to date and resources are the one listed here